src/resources/js/tribe-events-ajax-calendar.js

```diff
@@ -301,6 +301,15 @@
 					url = td.cur_url.split("?")[0];
 				}
 
+				// if using the shortcode
+				if ( $( document.getElementById( 'tribe-events' ) ).is( '.tribe-events-shortcode' ) ) {
```

Comment:
I think there is a `$wrapper` variable that holds the same reference to this DOM node so I don't think there's a need to have a new one.